cmd/verify/app/repository.go

```diff
@@ -386,10 +388,26 @@ func VerifyCmd(staged bool, repository string, rootFile string,
 	if err != nil {
 		return fmt.Errorf("error getting client state: %s", err)
 	}
-	targetFiles, err := c.Targets()
+
+	// Always try to retrieve the top-level targets.
```

Comment:
I believe this comment is wrong? The top-level targets are only verified if targets == nil?

Reply:
Oops, yes! I think I switched to doing top-level by default, or else the specified, rather than add on the specified. Thank you

Reply:
Ah no - sorry, let me clarify - the reason I did this is because this internally verifies the targets.json file, and I wanted to make sure the content in there was validated (the checksums and hashes). let me clarify that it is to silently verify the contents.